gpii/node_modules/solutionsRegistry/src/js/sr-validation-middleware.js

```diff
@@ -0,0 +1,180 @@
+/*
+
+    kettle.middleware "validators" designed to validate preference and settings payloads.
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+// Add the ability to require JSON5 files, the default usage of these grades will need to have this or they will fail to
+// expand the base schemas.
+require("json5/lib/register");
+
+fluid.registerNamespace("gpii.universal.solutionsRegistry.validators");
+
+fluid.require("%gpii-json-schema");
+
+/**
+ *
+ * A replacement for the default kettle validation middleware handle invokers backing function, which is more suitable
+ * for working with a schema that is generated based on the payload.
+ *
+ * @param {Object} that - The validation middleware component.
```

Comment:
A helpful practice we seem to be finding in other areas is to more specifically write {Component} or else even more informatively, {gpii.universal.solutionsRegistry.validators.base}

Reply:
Done throughout the solutionsRegistry sub-module.